nfv_tempest_plugin/tests/scenario/dcb_manager.py

```diff
@@ -0,0 +1,267 @@
+# Copyright 2018 Red Hat, Inc.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import re
+
+from nfv_tempest_plugin.tests.common import shell_utilities as shell_utils
+from oslo_log import log
+from tempest import config
+
+CONF = config.CONF
+LOG = log.getLogger('{} [-] nfv_plugin_test'.format(__name__))
+
+class DCBManager(object):
+
+    def create_and_apply_dcb_config(self, target_interface, dut_hypervisor):
+        """ Creates YAML file and applies the DCB config """
+
+        config_result = None
+        # Define the YAML template with a placeholder for the first_interface
+        yaml_template = """
```

Comment:
In general, I'd try to avoid hardcoded configurations.

Reply:
Changes done and uploaded again. Kindly check.